.github/usdchecker.py

```diff
@@ -120,6 +120,10 @@ def main():
     warnings = checker.GetWarnings()
     errors = checker.GetErrors()
     failedChecks = checker.GetFailedChecks()
+
+    # Skip 'missing' .mtlx documents, until usd-core includes UsdMtlX
+    if failedChecks:
+        failedChecks = [check for check in failedChecks if '.mtlx' not in check]
```

Comment:
Personally I don't like the coarseness of this check as errors from other rules can get ignored. I would use a regex that additionally contains part of the error string, but it probably does not matter in practice.

Reply:
Thanks for the suggestions. I'm not regex-savy enough to implement it that way; but since this is just a temporary measure until the USD package on PyPi includes UsdMtlX, are you okay if it's accepted as-is?

Reply:
Sure 👍

(on a second note: just doing a second substring check would be just as easy)